packages/imperative/src/imperative/src/plugins/utilities/npm-interface/install.ts

```diff
@@ -24,6 +24,40 @@ import { PluginManagementFacility } from "../../PluginManagementFacility";
 import { ConfigurationLoader } from "../../../ConfigurationLoader";
 import { UpdateImpConfig } from "../../../UpdateImpConfig";
 import { CredentialManagerOverride, ICredentialManagerNameMap } from "../../../../../security";
+import { IProfileTypeConfiguration } from "../../../../../profiles";
+import * as semver from "semver";
+import { ProfileInfo } from "../../../../../config";
+import { IExtendersJsonOpts } from "../../../../../config/src/doc/IExtenderOpts";
+
+// Helper function to update extenders.json object during plugin install.
+// Returns true if the object was updated, and false otherwise
+const updateExtendersJson = (
```

Comment:
Curious why this functions wasn't exported but its counterpart (in the uninstall.ts file) was exported and defined as a function?